src/Jackett.Common/Indexers/Norbits.cs

```diff
@@ -398,19 +375,19 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                             MinimumSeedTime = 172800 // 48 hours
                         };
 
-                        var genres = tRow.Find("span.genres").Text();
+                        var genres = row.QuerySelector("span.genres").TextContent;
```

Comment:
Progress, but no banana quite yet.

some rows do not contain any genres.
line 378 is generating
```
2020-03-01 18:02:03 | Error | Jackett.Common.IndexerException: Exception (norbits): Object reference not set to an instance of an object.  ---> System.Exception: Parse error  ---> System.NullReferenceException: Object reference not set to an instance of an object.    at Jackett.Common.Indexers.Norbits.PerformQuery(TorznabQuery query) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\Norbits.cs:line 378    --- End of inner exception stack trace ---    at Jackett.Common.Indexers.BaseWebIndexer.OnParseError(String results, Exception ex) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:line 830    at Jackett.Common.Indexers.Norbits.PerformQuery(TorznabQuery query) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\Norbits.cs:line 402    at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:line 323    --- End of inner exception stack trace ---    at Jackett.Common.Indexers.BaseIndexer.ResultsForQuery(TorznabQuery query) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:line 343    at Jackett.Common.Indexers.BaseWebIndexer.ResultsForQuery(TorznabQuery query) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\BaseIndexer.cs:line 803    at Jackett.Server.Controllers.ResultsController.Results(ApiSearch requestt) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Server\Controllers\ResultsController.cs:line 224
-- | -- | --
2020-03-01 18:02:02 | Error | Error on 20200301180202 for Norbits.txtSystem.NullReferenceException: Object reference not set to an instance of an object.    at Jackett.Common.Indexers.Norbits.PerformQuery(TorznabQuery query) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\Norbits.cs:line 378     Error, unable to parse result     at Jackett.Common.Indexers.Norbits.PerformQuery(TorznabQuery query) in C:\Users\Garfield69\Documents\GitHub\Jackett\src\Jackett.Common\Indexers\Norbits.cs:line 378
```
I think you first have to `var genres = row.QuerySelector("span.genres")` then check !null and then use `.TextContent` ?

Reply:
```suggestion
                        var genres = row.QuerySelector("span.genres")?.TextContent;
```

Reply:
That checks for null, and the if on the next line shows that this one should have been checking for null in the first place.